keyboards/smithrune/magnus/readme.md

```diff
@@ -0,0 +1,35 @@
+# Smith and Rune Magnus QMK folder
+
+![magnus](https://smithrune.com/cdn/shop/products/15-SR.jpg)
```

Comment:
Please rehost this on a dedicated image hosting site (imgur, etc)

Reply:
Done in 618decd983